larq/quantizers_test.py

```diff
@@ -49,6 +60,22 @@ def test_invalid_usage(self):
         with pytest.raises(ValueError):
             lq.quantizers.get("unknown")
 
+    def test_layer_as_input_quantizer(self):
+        input_data = testing_utils.random_input((1, 10))
+        model = tf.keras.Sequential(
+            [lq.layers.QuantDense(1, input_quantizer=DummyTrainableQuantizer())]
+        )
+        model.compile(optimizer="sgd", loss="mse")  # TODO modes?
+        model.fit(input_data, np.ones((1,)), epochs=1)
+
+    def test_layer_as_kernel_quantizer(self):
+        input_data = testing_utils.random_input((1, 10))
+        model = tf.keras.Sequential(
+            [lq.layers.QuantDense(1, kernel_quantizer=DummyTrainableQuantizer())]
+        )
+        model.compile(optimizer="sgd", loss="mse")  # TODO modes?
+        model.fit(input_data, np.ones((1,)), epochs=1)
```

Comment:
I think it would be good to check if `dummy_weight` is in `model.trainable_variables` here.

Reply:
Added in f44f187.